src/Authorization/Helpers/EventLogHelper.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Constants;
+using Altinn.Authorization.ABAC.Interface;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Constants;
+using Altinn.Platform.Authorization.Models;
+using Altinn.Platform.Authorization.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Documents.SystemFunctions;
+using Microsoft.FeatureManagement;
+
+namespace Altinn.Platform.Authorization.Helpers
+{
+    /// <summary>
+    /// Helper class for event logging
+    /// </summary>
+    public static class EventLogHelper
+    {
+        /// <summary>
+        /// Creates an authorization event
+        /// </summary>
+        /// <param name="featureManager">handler for feature manager service</param>
+        /// <param name="eventLog">handler for eventlog service</param>
+        public async static Task CreateAuthorizationEvent(IFeatureManager featureManager, IEventLog eventLog, XacmlContextRequest contextRequest, HttpContext context, XacmlContextResponse contextResponse)
+        {
+            if (await featureManager.IsEnabledAsync(FeatureFlags.AuditLog))
```

Comment:
Does this belong in a static helper class? To me this is a service implementations responsibility.